website/static/install

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+
+set -eu
+
+INSTALLPATH=/usr/local/bin
```

Comment:
I would make this user settable: 

for eg, many folks might not `~/.bin` or `~/.local/bin`: eg

`curl -SL https://raw.githubusercontent.com/siderolabs/talos/6f99922f681522066bc6aaae40efdd9ec728cc2d/website/static/install | INSTALLPATH=/tmp sh`

```suggestion
INSTALLPATH=${INSTALLPATH:-/usr/local/bin}
```